.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+version: 2
+
+test-defaults: &test-defaults
+  working_directory: ~/graphql-directive-uid
+  steps:
+    - attach_workspace:
+        at: ~/graphql-directive-uid
+    - run: |
+        yarn lint
+        yarn test
+
+jobs:
+  install-dependencies:
+    working_directory: ~/graphql-directive-uid
+    docker:
+      - image: circleci/node:6
```

Comment:
As long as we don't have any deps with native bindings I'm ok with that, otherwise builds could fail when ran on other versions of Node